### PR TITLE
feat : added functionality of create post

### PR DESCRIPTION
**Pull request by mariyasada**
-added functionality related to post
-user can create posts, edit posts, and remove posts (from the profile page).

### CODE REVIEW

**Review by munjalSaksham71 — APPROVED** [review]
Code Looks understandable. 

Just one suggestion:  You can reverse the array of posts to get the latest added post at the top,

File: src/Services/createpostservices/createpostservice.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+export const createPostService= async(postData,authenticationToken)=>{
+    console.log(postData);
+    console.log(authenticationToken);
+    try {
+        const {data,status}=await axios.post("/api/posts",{postData},{headers:{
+            authorization: authenticationToken
+            
+        }})
+        console.log(data,status);
```
Comment (Bhakti199) [inline src/Services/createpostservices/createpostservice.js]: remove extra console.log

File: src/Services/createpostservices/editpostservice.js [hunk 2]
```diff
@@ -0,0 +1,16 @@
+import axios from "axios";
+
+export const editPostService= async(postData,authenticationToken)=>{
+    try {
+        const {data,status}=await axios.post(`/api/posts/edit/${postData._id}`,{postData},{headers:{
+            authorization: authenticationToken
+            
+        }})
+        console.log(data,status);
```
Comment (Bhakti199) [inline src/Services/createpostservices/editpostservice.js]: here also

**Review by Bhakti199 — APPROVED** [review]
This code looks fine to me.
well written and structured.
this pr is good to merge.

**Pull request merged**